code/modules/surgery/bodyparts/bodyparts.dm

```diff
@@ -19,7 +19,7 @@
 
 	var/is_husked = FALSE //Duh
 	var/limb_id = SPECIES_HUMAN //This is effectively the icon_state for limbs.
-	var/limb_gender //Defines what sprite the limb should use if it is also sexually dimorphic.
+	var/limb_gender = "m" //Defines what sprite the limb should use if it is also sexually dimorphic.
```

Comment:
use define

Reply:
MALE/FEMALE are native byond constants for "male" and "female". limbs use "m"/"f" to cut down on icon_state length